Tools/MiniBrowser/haiku/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+SET(CMAKE_BUILD_TYPE Debug)
```

Comment:
I had removed this from the previous commits already. Setting the global variable here is a bad idea. You can set it on the target if you really need to debug MiniBrowser, but I suspect it will be of little help now that the UI is in place.